src/core/dash_internal.h

```diff
@@ -1718,6 +1718,10 @@ auto Segment<Key, Value, Policy>::BumpUp(uint8_t bid, SlotId slot, Hash_t key_ha
     return Iterator{bid, slot};
   }
 
+  if (swap_bid == bid) {
```

Comment:
how can swap_bid == bid?
in this flow bid is a stash bucket? why would we get to swap inside stash?

Reply:
Uh, I got confused with `target_bid`. Indeed `bid` is a stash bucket and this equality should never be true under any flow. Will remove this.